### PR TITLE
{include,src}/requests/* --> {include,src}/supernode/requests/

### PR DESCRIPTION
**Pull request by cybay**
…ernode/requests/'

### CODE REVIEW

File: test/fixture.h [hunk 1]
```diff
@@ -148,6 +151,7 @@ class GraftServerTestBase : public ::testing::Test
         std::atomic<graft::HttpConnectionManager*> phttpcm{nullptr};
     private:
         std::thread th;
+        graft::supernode::system_info::Counter sys_info_;
```
Comment (bitkis) [inline test/fixture.h]: can we avoid pushing this changes until we agree on them?

File: test/json_rpc_test.cpp [hunk 2]
```diff
@@ -160,6 +163,8 @@ struct JsonRpcTest : public ::testing::Test
 
     void startServer()
     {
+        graft::supernode::system_info::Counter sys_info;
```
Comment (bitkis) [inline test/json_rpc_test.cpp]: same as above

Comment (cybay) [reply test/json_rpc_test.cpp]: Will be done at the next step

File: test/fixture.h (hunk 1 above)
Comment (cybay) [reply test/fixture.h]: We can come to better solution later. But now it give us working tests.

File: include/supernode/requests/authorize_rta_tx.h [hunk 3]
```diff
@@ -1,9 +1,7 @@
-#ifndef AUTHORIZERTATXREQUEST_H
-#define AUTHORIZERTATXREQUEST_H
 
-#include "router.h"
-#include "inout.h"
+#pragma once
 
+#include "router.h"
 
 namespace graft {
```
[the remark this answers is not in the document]
Comment (cybay) [reply include/supernode/requests/authorize_rta_tx.h]: Will be done at the next step

**Comment by bitkis** [comment]
After discussing it with Andrew: this is a preliminary commit he needs to move forward

**Pull request merged**